src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -305,9 +310,6 @@ public function supportsDenormalization(mixed $data, string $type, string $forma
         return class_exists($type) || (interface_exists($type, false) && null !== $this->classDiscriminatorResolver?->getMappingForClass($type));
     }
 
-    /**
-     * @return mixed
-     */
```

Comment:
This must not be removed yet

Reply:
It was a fabbot.io suggestion, but I'll revert it.